### ISSUE TITLE
Why don't use the particle in joomla content?

### ISSUE THREAD

**Issue by emanuelepaccamiccio** [gantry/gantry5#1003]
I'm really excited about using Gantry for my web projects , really congratulations for the work you have done . I think you have introduced new ways to build templates . When I realized the power of the particle and see which are available as a module or as object to attach to a menu item I wondered ... why not have it available when building content with content based on joomla ?

**Comment by mahagr** [comment]
Particles are already available as main page content; Just create/edit any menu item with Gantry 5 Themes / Custom Page and click on Options tag. You can also add particles into the articles just like any modules, but for that you need to create those modules separately.

I do agree that we are still missing a button in editor to add particles directly from the editor...

**Comment by emanuelepaccamiccio** [comment]
Yes, a button in the editor would be a really interesting and would allow a custom layout of content

**Comment by mahagr** [comment]
Requires #1005 to be implemented first.

**Comment by BjoernH01** [comment]
But there is a HUGE difference in speed of work and work flow if you can just put in a particle in an article while writing instead of leaving the article, making the module / particle and then loading the particle in the article...

**Comment by emanuelepaccamiccio** [comment]
I would be a fantastic thing to have the opportunity to build a custom content by selecting a particle from an article of Joomla and make the content based joomla very smart ! The current limit of the particle is that it isn't  dynamic publishing tool , but it could become one !!!

**Comment by joomlabeat** [comment]
Any update on the Particle XTD Editor button, so we can create/edit particles directly from inside the editor? That would be one of the greatest additions for Gantry.
